x-pack/plugins/enterprise_search/public/applications/workplace_search/components/overview/overview.tsx

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useContext, useEffect, useState } from 'react';
+import { EuiPage, EuiPageBody, EuiSpacer } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import { SetWorkplaceSearchBreadcrumbs as SetBreadcrumbs } from '../../../shared/kibana_breadcrumbs';
+import { SendWorkplaceSearchTelemetry as SendTelemetry } from '../../../shared/telemetry';
+import { KibanaContext, IKibanaContext } from '../../../index';
+
+import { IAccount } from '../../types';
+
+import { ErrorState } from '../error_state';
+
+import { Loading } from '../shared/loading';
+import { ProductButton } from '../shared/product_button';
+import { ViewContentHeader } from '../shared/view_content_header';
+
+import { OnboardingSteps } from './onboarding_steps';
+import { OrganizationStats } from './organization_stats';
+import { RecentActivity, IFeedActivity } from './recent_activity';
+
+export interface IAppServerData {
+  hasUsers: boolean;
+  hasOrgSources: boolean;
+  canCreateContentSources: boolean;
+  canCreateInvitations: boolean;
+  isOldAccount: boolean;
+  sourcesCount: number;
+  pendingInvitationsCount: number;
+  accountsCount: number;
+  personalSourcesCount: number;
+  activityFeed: IFeedActivity[];
+  organization: {
+    name: string;
+    defaultOrgName: string;
+  };
+  isFederatedAuth: boolean;
+  currentUser: {
+    firstName: string;
+    email: string;
+    name: string;
+    color: string;
+  };
+  fpAccount: IAccount;
+}
+
+export const defaultServerData = {
+  accountsCount: 1,
+  activityFeed: [],
+  canCreateContentSources: true,
+  canCreateInvitations: true,
+  currentUser: {
+    firstName: '',
+    email: '',
+    name: '',
+    color: '',
+  },
```

Comment:
I'm wondering if we should just leave these out if they're not being used in the MVP? We risk leaving less leftover cruft from ent-search if we port on an as-needed basis